InstabugSample/src/screens/SettingsScreen.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import { Switch, StyleSheet, Text, View } from 'react-native';
+import Instabug, { BugReporting } from 'instabug-reactnative';
+
+import Button from '../components/Button';
+import ColorButton from '../components/ColorButton';
+import Screen from '../components/Screen';
+import Section from '../components/Section';
+
+const colors = ['crimson', 'olivedrab', 'cornflowerblue', 'gold'];
+
+function SettingsScreen() {
+  const [isLightMode, setIsLightMode] = useState(true);
```

Comment:
**suggestion:** Changing this state from `boolean` to `Instabug.colorTheme` have some pros IMO:

1. Consistency along with `currentColor` state.
    ```js
    const [currentTheme, setCurrentTheme] = useState(Instabug.colorTheme.light);
    ```
1. Component-independent: can work with other components as `Select`.